### DISCUSSION TITLE
I don't seem be getting the userHandle in the response anymore when calling startAuthentication()

### DISCUSSION THREAD

**Discussion by treeder** [MasterKale/SimpleWebAuthn#561] [Troubleshooting]
I used to get a response like the following: https://github.com/MasterKale/SimpleWebAuthn/issues/490#issuecomment-1859430975

But now I'm not getting the `userHandle` field in there. I don't know what changed or how to fix it and I can't find it in the docs anywhere. I'm setting the userID and userName fields during registration, but can't see how to get them back anymore when logging in. 

Any tips?

**Comment by MasterKale** [comment]
Hello @treeder, can you help me understand your current setup? What version(s) of SimpleWebAuthn are you using? And how about OS, browser, and versions of each?

And under what configuration did you last get `userHandle` for whatever credential you want to focus on here?

**Reply by treeder** [reply]
I honestly don't know what changed. I've tried to go back through everything, but no luck. 

Is there a way to check that the user ID was set properly when creating the pass key? (assuming that's where userHandle comes from).

**Reply by treeder** [reply]
Oh, and I upgraded to v10 to see if it would fix this (it didn't). Change the userID to the arraybuffer and all that too.

**Reply by treeder** [reply]
@MasterKale do you have any examples of using `userHandle`?

**Reply by treeder** [reply]
Ok, so I deleted all my passkey for the site and recreated it and of course that fixed it. So I think I'm good now. 

I was expecting that if I created a new passkey with the same user.id and everything that it would overwrite an existing one, but maybe that's not the case?

**Reply by MasterKale** [reply]
So if you didn't change anything about your code, and didn't upgrade SimpleWebAuthn, did you update your OS? Perhaps your browser? Something external must have changed if you stopped getting `userHandle`...

**Reply by treeder** [reply]
I really have no idea, I probably made some passkey while [testing out all the different options](https://github.com/MasterKale/SimpleWebAuthn/discussions/539) that wouldn't go away even when I created new ones. 🤷
